scripts/us_epa/superfund/sites/tar_creek/process_report2020.py

```diff
@@ -0,0 +1,208 @@
+import os
```

Comment:
please add module level comment and copyright

Reply:
I have added the copyright and module-level comment